### ISSUE TITLE
Parametrize mark returning error. 

### ISSUE THREAD

**Issue by colbykbarton-digicert** [taverntesting/tavern#377]
```
test_name: example

strict: False 

marks:
  - parametrize:
      key: invalid_characters
      vals:
        - "!"
        - "@"
        - "#"
        - "$"
        - "%"
        - "^"
        - "*"
        - "("
        - ")"
        - "~"
        - "?"
        - ">"
        - "<"
        - "/"
        - "\\"
  - usefixtures:
      - get_org_name
      - get_org_name_long
  - stages:
  - name: Creating Organization (Invalid Characters)
    request:
      url: "{base_url}/organization"
      method: POST
      headers:
        X-DC-DEVKEY: "{api_key_admin1}"
      json:
        name: "{invalid_characters} {get_org_name}"
        country: US
        address: 1234 neat place st
        city: Houston
        state: Texas
        zip: 77001
        telephone: 801-888-8989
        container:
          id: "{container_id1}"
        organization_contact:
          first_name : new_contact_first
          last_name : new_contact_last
          job_title : CEO
          email : first.last@neworg.com
          telephone : 801-888-8899
          telephone_extension : 465
    response:
      status_code: 400
```

This throws an error of ```TypeError: _parse_parametrize_args() missing 1 required positional argument: 'argvalues'```

I have tried running even simpler parametrized tests but get the same error every time.

**Comment by colbykbarton-digicert** [comment]
We have tried running this on tavern-0.26.3 and it works fine, the bug seems to be in tavern-0.26.4

**Comment by z4c** [comment]
seems to be more related to pytest 4.6.x ( first jun 2019 release : https://pypi.org/project/pytest/#history )

**Comment by colbykbarton-digicert** [comment]
Thanks @z4c that is correct.

**Comment by rmaughan** [comment]
Encountering the same issue - and confirmed this is resolved by rolling back to pytest 4.5.0.  Any ideas on when latest version of pytest 4.6.x or 5.x.x might be supported with Tavern?

**Comment by fabiobento512** [comment]
I am also having this same issue. Reverting to pytest 4.5.0 fixed the problem as @rmaughan mentioned. Can you fix this so tavern works with the latest pytest (or if it is a problem in pytest, can you contact them?)?

**Comment by michaelboulton** [comment]
This is fixed in 0.26.5+ but the use of fixtures needs updating which might be a significant chunk of work

https://github.com/taverntesting/tavern/issues/367
